gradle/libs.versions.toml

```diff
@@ -97,6 +98,7 @@ arrow-vector = { module = "org.apache.arrow:arrow-vector", version.ref = "arrow"
 avro-avro = { module = "org.apache.avro:avro", version.ref = "avro" }
 awssdk-bom = { module = "software.amazon.awssdk:bom", version.ref = "awssdk-bom" }
 awssdk-s3accessgrants = { module = "software.amazon.s3.accessgrants:aws-s3-accessgrants-java-plugin", version.ref = "awssdk-s3accessgrants" }
+awssdk-crt = { module = "software.amazon.awssdk.crt:aws-crt", version.ref = "awssdk-crt" }
```

Comment:
why is this one needed now?

Reply:
I've addressed this above and removed now.